src/Http.hs

```diff
@@ -0,0 +1,21 @@
+{-# LANGUAGE OverloadedStrings #-}
+module Http (start) where
+
+import           Data.String
+import           Control.Monad
+import           Control.Concurrent
+import           Network.Wai
+import           Network.HTTP.Types
+import           Network.Wai.Handler.Warp (run)
+
+start :: IO String -> IO ()
+start trigger = do
+  putStrLn $ "http://localhost:" ++ show port
```

Comment:
I would rather have this being printed after the server actually starts to listen. Like here: https://github.com/zalora/kraken/blob/master/src%2FNetwork%2FWai%2FHandler%2FWarp%2FRun.hs#L23. I also would prefer the wording `listening on http://localhost:PORT`.